user-apps/vsockproxy/default.nix

```diff
@@ -0,0 +1,33 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  stdenv,
+  pkgs,
+  lib,
+  ...
+}:
+stdenv.mkDerivation {
+  name = "vsockproxy";
+
+  nativeBuildInputs = with pkgs; [meson ninja];
+
+  src = pkgs.fetchFromGitHub {
+    owner = "nesteroff";
```

Comment:
I think we should create a repo inside TIIUAE for this

Reply:
Ok, now it's in tiiuae.